opendatahub-io/odh-dashboard
# Fix Pipeline Job titles are partially handled

Closes: #1242

## Description
#1242
It fixes "_When a Pipeline Scheduled Run (aka a Job) creates Triggered Runs it creates it with a rather unpleasant name -- a fully lowercase, concatenated together string of the name of the Job._"

### Multiple runs delete
<img width="1205" alt="Screenshot 2023-07-14 at 4 54 41 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/2117670/e3ec2149-21a6-4ef2-a8a3-57529c9293f7">

### Single run delete
<img width="1378" alt="Screenshot 2023-07-14 at 6 07 58 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/2117670/0fefb49d-947d-454e-884e-2401160c8fa3">


### Run details - including bottom drawer
<img width="1614" alt="Screenshot 2023-07-14 at 4 54 04 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/2117670/99806521-ace8-4104-8eb7-8579d42c55c8">





## How Has This Been Tested?

- Create scheduled run
- Go to triggered tab once scheduled run triggered
- Select multiple or single triggered run to delete run(s), on the delete modal the formatted  name(s)  should appear
- Click on the triggered run to go to the run details, see the formatted name on the page title and the bottom drawer


## Test Impact



## Request review criteria:



Self checklist (all need to be checked):

If you have UI changes: 


After the PR is posted & before it merges:
